evmos/chain.json

```
@@ -179,11 +173,34 @@
         },
         "ibc_go_version": "v7.3.1",
         "binaries": {
-          "linux/amd64": "https://github.com/evmos/evmos/releases/download/v16.0.3/evmos_16.0.3_Linux_amd64.tar.gz",
-          "linux/arm64": "https://github.com/evmos/evmos/releases/download/v16.0.3/evmos_16.0.3_Linux_arm64.tar.gz",
-          "darwin/amd64": "https://github.com/evmos/evmos/releases/download/v16.0.3/evmos_16.0.3_Darwin_amd64.tar.gz",
-          "darwin/arm64": "https://github.com/evmos/evmos/releases/download/v16.0.3/evmos_16.0.3_Darwin_arm64.tar.gz",
-          "windows/amd64": "https://github.com/evmos/evmos/releases/download/v16.0.3/evmos_16.0.3_Windows_amd64.zip"
+          "linux/amd64": "https://github.com/evmos/evmos/releases/download/v16.0.4/evmos_16.0.4_Linux_amd64.tar.gz",
+          "linux/arm64": "https://github.com/evmos/evmos/releases/download/v16.0.4/evmos_16.0.4_Linux_arm64.tar.gz",
+          "darwin/amd64": "https://github.com/evmos/evmos/releases/download/v16.0.4/evmos_16.0.4_Darwin_amd64.tar.gz",
+          "darwin/arm64": "https://github.com/evmos/evmos/releases/download/v16.0.4/evmos_16.0.4_Darwin_arm64.tar.gz",
+          "windows/amd64": "https://github.com/evmos/evmos/releases/download/v16.0.4/evmos_16.0.4_Windows_amd64.zip"
+        },
+        "next_version_name": "v17.0.0"
+      },
+      {
+        "name": "v17.0.0",
+        "tag": "v17.0.0",
+        "height": 20101000,
+        "recommended_version": "v17.0.0",
+        "compatible_versions": [
+          "v17.0.0"
+        ],
+        "cosmos_sdk_version": "evmos/cosmos-sdk v0.47.8-evmos",
```

ditto this is still v0.47.5-evmos.2

```suggestion
        "cosmos_sdk_version": "evmos/cosmos-sdk v0.47.5-evmos.2",
```